docs/track.rst

```diff
@@ -427,6 +429,16 @@ Example::
               ]
             }
           }
+        ],
+        "on-failure": [
```

Comment:
There is a syntax error in the pipeline. The element is called `on_failure`instead of `on-failure` (see [docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/handling-failure-in-pipelines.html)).